### PR TITLE
Fix code coverage reporting on GitHub actions

### PR DESCRIPTION
**Pull request by JoelSpeed**
## Description

There were a couple of issues that prevented the code climate code coverage reports from uploading. I think this should be resolved now based on the checks on this PR.

## Motivation and Context

Fix code climate uploads.

## How Has This Been Tested?

Looking at test results on this PR

## Checklist:

- [x] My change requires a change to the documentation or CHANGELOG.
- [x] I have updated the documentation/CHANGELOG accordingly.
- [x] I have created a feature (non-master) branch for my PR.

CC @shinebayar-g

### CODE REVIEW

File: .github/workflows/test.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # manually exiting from script, because after-build needs to run always
 set +e
 
-if [ -z $CC_TEST_REPORT_ID ]; then
-  echo "1. CC_TEST_REPORT_ID is unset, skipping"
+if [ -z $CC_TEST_REPORTER_ID ]; then
```
Comment (shinebayar-g) [inline .github/workflows/test.sh:5]: Oh :sweat_smile: :facepalm:

Comment (JoelSpeed) [reply .github/workflows/test.sh:5]: It makes much more sense now right 😅  I missed this too

Comment (shinebayar-g) [reply .github/workflows/test.sh:5]: Idk how I came up with such variable at first place. I guess I didn't just copy paste.

Comment (JoelSpeed) [reply .github/workflows/test.sh:5]: I don't know if I suggested it on a comment or something maybe?

Comment (shinebayar-g) [reply .github/workflows/test.sh:5]: Actually you gave the right variable name `CC_TEST_REPORTER_ID` . It was my mistake.

**Review by NickMeves — APPROVED** [review]

**Pull request merged**